rskj-core/src/main/java/co/rsk/util/JacksonParserUtil.java

```diff
@@ -0,0 +1,55 @@
+package co.rsk.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public final class JacksonParserUtil {
```

Comment:
it looks like that only `mapper.treeToValue()` allows `null`'s (returns null if null is provided). The others ( `readTree()` ones ) do throw an exception if provided argument is `null`.

So may be ( as @illuque-iov mentioned in the other comment ) we don't need this new class and can handle null's on the case-by-case basis ( sees we have 2 or 3 places )

Reply:
I don't think so @Vovchyk, they have this new NullNode and MissingNode clases that inherits from JsonNode for `readTree` so exception are not going to be thrown for null and empty string content.